t/tokenize/hashes.xml

```diff
@@ -21,4 +21,38 @@
       <p>Deferred tokenize: One hash: #, not two; [expanded a].</p>
     </para>
   </section>
+  <section inlist="toc" xml:id="S2">
+    <tags>
+      <tag>2</tag>
+      <tag role="refnum">2</tag>
+      <tag role="typerefnum">§2</tag>
+    </tags>
+    <title><tag close=" ">2</tag>In definitions</title>
+    <para xml:id="S2.p1">
+      <p>Also two hashes##, not one</p>
+    </para>
+    <para xml:id="S2.p2">
+      <p>Also two hashes##, not one</p>
+    </para>
+  </section>
+  <section inlist="toc" xml:id="S3">
+    <tags>
+      <tag>3</tag>
+      <tag role="refnum">3</tag>
+      <tag role="typerefnum">§3</tag>
+    </tags>
+    <title><tag close=" ">3</tag>File io</title>
+    <para xml:id="S3.p1">
+      <p>Reading file badchars.tex:<break/>
+Line:“Bad chars: ^b$@##˙x ! ”<break/>
+
+Line:“More Bad chars: ^b$@##˙x ! ”<break/>
+
+Line:“And More Bad chars: ^b$@##˙x ! ”<break/>
+
+Line:““par ”<break/>
+
+Full Contents:“Bad chars: ^b$@##˙x ! More Bad chars: ^b$@##˙x ! And More Bad chars: ^b$@##˙x ! “par ”<break/></p>
```

Comment:
The test matches the PDF nicely. I'm only doubting my eyes a little with the `Full Contents` spacing at the ends of the lines - the PDF seems to have an extra space after the exclamation chars. But that may be fixed by the other PR?

Looks good!

Reply:
Hmm, is that two spaces, or just a wide space (due to punctuation?). Funny, I can't see that at all!!